hhs_oauth_server/settings/base.py

```diff
@@ -466,7 +466,8 @@
 # Since this is internal False may be acceptable.
 SLS_VERIFY_SSL = env('DJANGO_SLS_VERIFY_SSL', False)
 
-AUTHENTICATION_BACKENDS = ('django.contrib.auth.backends.ModelBackend',)
+AUTHENTICATION_BACKENDS = ('apps.accounts.email_auth_backend.EmailBackend',
+                           'django.contrib.auth.backends.ModelBackend')
```

Comment:
Is there a reason to keep ModelBackend here? It seems like the new EmailBackend is completely taking over its job.

Reply:
Trying to make minimal changes.  + we still use this for admin access